.github/workflows/main.yml

```diff
@@ -0,0 +1,125 @@
+name: main
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+  pull_request:
```

Comment:
Maybe add `types: [ready_for_review]` to avoid the multiple triggers?

Reply:
I fixed the double trigger in my last commit by saying that the "push" should trigger the workflow only on master